web/components/ConferenceHeader/ConferenceHeader.tsx

```diff
@@ -29,9 +29,8 @@ export const ConferenceHeader = ({
       </h1>
       <div className={styles.summary}>
         <div className={styles.datesAndHostedByWrapper}>
-          <p className={styles.dates}>
-            <DateRange startTimestamp={startDate} endTimestamp={endDate} />
-          </p>
```

Comment:
Unused props and imports here now; these can also be removed from the Sanity query now and [[...slug]].tsx can stop passing these to the component.

Reply:
Good point! I figured I'd leave the DateRange implementation as-is because some version of it might be needed in the future once the temporary hardcoding can be fixed, but at the very least it makes complete sense not to spend bandwidth pointlessly fetching the unused data. And I should've removed the import as well.

- Import removed in 691eba3a1a3f1a1fbf3c7a418b2853c596d233ee
- startDate/endDate removed from query/props in 9bc9f224e58553b4cc3e04f7323f0fa01aa69661